server/container_list.go

```diff
@@ -40,7 +40,7 @@ func (s *Server) ListContainers(ctx context.Context, req *pb.ListContainersReque
 	if filter.Id != "" {
 		id, err := s.CtrIDIndex().Get(filter.Id)
 		if err != nil {
-			return nil, err
```

Comment:
log this error with logrus please

Reply:
and add a comment why we skip returning the error

Reply:
fixed

Reply:
this doesn't look fixed